components/dashboard/sections/contributions/Contributions.tsx

```diff
@@ -444,72 +634,188 @@ const Contributions = ({ accountSlug, direction }: ContributionsProps) => {
+                  </span>
+                  <PlusIcon size={20} />
+                </Button>
+                {showCreatePendingOrderModal && (
+                  <CreatePendingContributionModal
+                    hostSlug={accountSlug}
+                    onClose={() => setShowCreatePendingOrderModal(false)}
+                    onSuccess={() => {
+                      refetch();
+                      refetchMetadata();
+                    }}
+                  />
+                )}
+              </React.Fragment>
+            ) : null
+          }
         />
-      )}
+        <Filterbar {...queryFilter} />
 
-      {error ? (
-        <MessageBoxGraphqlError error={error} />
-      ) : !loading && selectedOrders.length === 0 ? (
-        <EmptyResults
-          entityType="CONTRIBUTIONS"
-          hasFilters={queryFilter.hasFilters}
-          onResetFilters={() => queryFilter.resetFilters({})}
-        />
-      ) : (
-        <div className="flex flex-col gap-4">
-          <DataTable
-            loading={loading}
-            columns={columns}
-            data={selectedOrders}
-            mobileTableView
-            nbPlaceholders={nbPlaceholders}
+        {isIncoming && !onlyExpectedFunds && metadata?.account?.[ContributionsTab.PAUSED].totalCount > 0 && (
+          <PausedIncomingContributionsMessage
+            account={metadata.account}
+            count={metadata.account[ContributionsTab.PAUSED].totalCount}
+          />
+        )}
+
+        {error ? (
+          <MessageBoxGraphqlError error={error} />
+        ) : !loading && selectedOrders.length === 0 ? (
+          <EmptyResults
+            entityType="CONTRIBUTIONS"
+            hasFilters={queryFilter.hasFilters}
+            onResetFilters={() => queryFilter.resetFilters({})}
           />
-          <Pagination
-            totalPages={pages}
-            page={currentPage}
-            onChange={page => queryFilter.setFilter('offset', (page - 1) * limit)}
+        ) : (
+          <div className="flex flex-col gap-4">
+            <DataTable
+              loading={loading}
+              columns={columns}
+              data={selectedOrders}
+              mobileTableView
+              nbPlaceholders={nbPlaceholders}
+              onClickRow={row => onToogleOrderDrawer(row.original.legacyId)}
+            />
+            <Pagination
+              totalPages={pages}
+              page={currentPage}
+              onChange={page => queryFilter.setFilter('offset', (page - 1) * limit)}
+            />
+          </div>
+        )}
+        {editOrder.order && (
+          <EditOrderModal
+            accountSlug={accountSlug}
+            order={editOrder.order}
+            action={editOrder.action}
+            onClose={() => setEditOrder({ order: null, action: null })}
           />
-        </div>
+        )}
+      </div>
+      {LoggedInUser?.hasPreviewFeatureEnabled(PREVIEW_FEATURE_KEYS.DASHBOARD_CONTRIBUTION_DETAILS) && (
```

Comment:
I don't get this check. Will the drawer just fail to open if the preview feature is off?